src/main/core/worker.rs

```diff
@@ -499,6 +506,13 @@ mod export {
         Worker::with_mut(|w| w.shared.increment_plugin_error_count()).unwrap()
     }
 
+    /// SAFETY: This is unsafe since we're returning a pointer for a reference that's not valid
```

Comment:
Nit - I think these conventionally such comments are phrased as describing what the caller's responsibility is when using this function to avoid unsoundness. e.g. "SAFETY: The returned pointer must not be accesses after this worker thread has exited"

Reply:
Updated.